napari_builtins/_tests/test_io.py

```diff
@@ -94,8 +93,8 @@ def test_zarr_with_unrelated_file(tmp_path):
     image = np.random.random((10, 20, 20))
     image_name = 'my_image'
     root_path = tmp_path / 'dataset.zarr'
-    grp = zarr.open(str(root_path), mode='a')
-    grp.create_dataset(image_name, data=image)
+    grp = zarr.open(store=str(root_path), mode='a')
+    grp.create_dataset(image_name, data=image, shape=image.shape)
```

Comment:
```suggestion
    grp.create_array(image_name, data=image, shape=image.shape)
```
See: https://github.com/napari/napari/pull/7215/files#r1810721505